.swiftlint.yml

```diff
@@ -25,3 +25,4 @@ file_header:
                     \/\/  Created by .*? on \d{1,2}\/\d{1,2}\/\d{2}\.
                     \/\/  Copyright © \d{4} Realm\. All rights reserved\.
                     \/\/
+line_length: 120
```

Comment:
I've added this here to prevent the `swiftlint` build phase to generate warnings (since it's using the installed version, which probably is the latest stable one).